translations/messages-en.properties

```diff
@@ -274,7 +274,7 @@ Error\ saving\ feedback = Error saving feedback
 feedback.submitted = Feedback submitted
 Error\ updating\ group = Error updating group
 Antenatal\ Care = Antenatal Care
-Stock\ Monitoring = Stock Monitoring
+Reporting\ Rates = Reporting Rates
```

Comment:
We shouldn't really rename keys. This causes problems with the POE integration. Read: https://github.com/medic/medic-docs/blob/master/md/dev/translations.md#modifying-or-removing-translation-keys

Also anyone who is using this in production may have customised the name which will now be reverted to the default.

I propose leaving the key as "Stock Monitoring" and just changing the value through POE.